src/components/IpfsInput/types.ts

```diff
@@ -0,0 +1,10 @@
+export interface IpfsInputprops {
+    /**
+     * Set theme  either 'textOnly' or 'withIcon'
+     */
+    Theme?: 'textOnly' | 'withIcon';
+    SaveToIpfs?: boolean;
+    FileName?: string;
+    Type?: string;
+    OnFinish: (value: any) => void;
+}
```

Comment:
1. Could you convert these names to camelCase ? SaveToIpfs => saveToIPFS
2. Please also add comments for each type since they are used by storybook.
3. i think `onComplete` is better naming than `onFinish`

Reply:
There is a event handler in SaveFile as OnComplete therefore added OnFinish